js/kafka/kafka-producer.js

```diff
@@ -18,6 +18,7 @@ export class KafkaProducer extends Producer {
         this.timeout = conf("KAFKA_PRODUCER_TIMEOUT", 30000);
         this.compression = conf("KAFKA_PRODUCER_COMPRESSION", null);
         this.maxInFlightRequests = conf("KAFKA_PRODUCER_MAX_INFLIGHT_REQUESTS", null);
+        this.globalDiffusion = conf("KAFKA_GLOBAL_DIFFUSION", true);

```

Comment:
This Global diffusion behavior will be generic for all future adapters, thus I believe we should name the var "GLOBAL_DIFFUSION" instead.